connect/runtime/src/test/java/org/apache/kafka/connect/integration/ConnectWorkerIntegrationTest.java

```diff
@@ -0,0 +1,178 @@
+/*
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.